src/main/frontend/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
FYI, GitHub wants a EOF newline

Reply:
This was an auto-generated file and I didn't touch it, but if I do I'll make sure it looks good.